config/routes.rb

```diff
@@ -1,9 +1,20 @@
 Rails.application.routes.draw do
   root 'users#index'
+
   resources :users, param: :name, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-    resources :comments, only: [:index, :show, :create, :update, :destroy]
-resources :likes, only: [:create, :destroy]
+    resources :posts, only: [:index, :show] do
+      resources :comments, only: [:create, :update, :destroy]
+
```

Comment:
![image](https://github.com/Kidd254/BlogApp/assets/93039359/abac34a5-4ebf-4978-9b24-449ec55fce47)

- [ ] Great work on configuring the comment creation but I am getting this error which you can see in the image above :arrow_up: Kindly fix it to make your project follow the requirements :+1: